requirements.txt

```diff
@@ -3,7 +3,7 @@ boto3==1.5.22
 coverage==4.5
 cryptography==2.1.4
 decorator==4.2.1
-Django==1.11.10
+Django==1.11.15
```

Comment:
these changes to requirements.txt are redundant, I have another PR open that is bumping them right now